docs/multi-vcenter.md

```diff
@@ -0,0 +1,39 @@
+# Multi VCenter support
```

Comment:
Is most of this better suited to Also should ahve some explanation how it works in https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/blob/main/docs/identity_management.md ?